### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: util/env_win.cc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // This file contains source that originates from:
 // http://code.google.com/p/leveldbwin/source/browse/trunk/win32_impl_src/env_win32.h
 // http://code.google.com/p/leveldbwin/source/browse/trunk/win32_impl_src/port_win32.cc
-// Those files dont' have any explict license headers but the 
+// Those files don't have any explict license headers but the
```
Comment (paveljanik) [inline util/env_win.cc]: explict -> explicit...

Comment (laanwj) [reply util/env_win.cc]: yes, please fix that one too

**Comment by ghost** [comment]
@paveljanik @laanwj Done.

**Comment by laanwj** [comment]
@mcccs thanks!

**Pull request merged**